### PR TITLE
Add minimal contract wrapper

### PR DESCRIPTION
**Pull request by tkmct**
Added
- deposit
- submit

TODO in future PR:
- Add README
- Error handling
- Add API
  - start_checkpoint
  - start_exit
  - finalize_exit
  - challenge
- Handle env values
  - contract address
  - predicate address

### CODE REVIEW

File: contract-wrapper/src/contract_wrapper.rs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            _web3: web3,
+            _eloop,
+            _address: address,
+            inner: contract,
+        }
+    }
+
+    pub fn deposit(&self, from: Address, amount: u64) -> Result<H256, Error> {
+        let predicate_address: Address =
+            "B3a39641f23aaaAB328B94b0fec83E1E4A08bc2B".parse().unwrap(); // TODO: fix predicate address
```
Comment (syuhei176) [inline contract-wrapper/src/contract_wrapper.rs]: This address stands for ownership predicate address now.

File: contract-wrapper/src/contract_wrapper.rs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+use ethabi::Token;
+use web3::contract::{Contract, Options};
+use web3::futures::Future;
+use web3::transports::{EventLoopHandle, Http};
+use web3::types::{Address, H256};
+
+type Error = ();
+
+pub struct ContractWrapper {
+    _eloop: EventLoopHandle,
```
Comment (syuhei176) [inline contract-wrapper/src/contract_wrapper.rs]: Does `ContractWrapper` object need to keep `_eloop`?

Comment (tkmct) [reply contract-wrapper/src/contract_wrapper.rs]: Yes, if _eloop is dropped, it starts to fail with `IOError: broken pipe`

File: contract-wrapper/src/contract_wrapper.rs (hunk 1 above)
Comment (tkmct) [reply contract-wrapper/src/contract_wrapper.rs]: @syuhei176 
ok! what do you think is a good way to inject predicate address?

Comment (syuhei176) [reply contract-wrapper/src/contract_wrapper.rs]: Yes, How about replace `amount: u64` to `stateObject: stateObject`?
And maybe you can get tuple by `stateObject.to_tuple()`.

Comment (tkmct) [reply contract-wrapper/src/contract_wrapper.rs]: That looks good

Comment (tkmct) [reply contract-wrapper/src/contract_wrapper.rs]: Fixed!

**Pull request merged**